Cargo.toml

```
@@ -28,7 +28,9 @@ all = "warn"
 const-hex = { version = "1.11.1", default-features = false }
 once_cell = "1.19.0"
 tiny-keccak = { version = "2.0.2", features = ["keccak"] }
-stylus-sdk = { version = "0.8.3-rc.1", default-features = false }
+stylus-sdk = { version = "0.8.3", default-features = false, features = [
```

```suggestion
stylus-sdk = { version = "=0.8.3", default-features = false, features = [
```
Should we also fix the version as we do for [contracts](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/605/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R83)?